api/api-tag-overview.md

```diff
@@ -0,0 +1,24 @@
+# API Reference Tag Overview
+
+Timescale Documentation utilizes a tagging system to describe functions in the API Reference section. These tags include "Community", "Experimental", "Toolkit", and "Experimental" (for TimescaleDB Toolkit functions). Below is more information on what each tag represents.
```

Comment:
```suggestion
The Timescale API Reference uses tags to categorize functions. The tags are `Community`, `Experimental`, `Toolkit`, and `Experimental (Toolkit)`. This section explains each tag.
```

Have to say, the interplay between Experimental and Toolkit and Experimental (Toolkit) is _super_ confusing.